data/benefits/javascript/region-corse-bourse-a-la-mobilite-mobighjovani.yml

```
@@ -23,8 +26,9 @@ conditions_generales:
     value: 18500
     period: year
 profils: []
-link: https://www.isula.corsica/jeunesse/Bourse-Mobighjovani-300-pour-les-jeunes-qui-souhaitent-voyager_a276.html
-instructions: https://www.isula.corsica/jeunesse/Bourse-Mobighjovani-300-pour-les-jeunes-qui-souhaitent-voyager_a276.html
+link: https://www.ghjuventu.corsica/articles/444E7A9F-9238-4011-9112-A87F95C51571
+instructions: https://www.ghjuventu.corsica/articles/444E7A9F-9238-4011-9112-A87F95C51571
+teleservice : https://www.ghjuventu.corsica/my/dashboard
```

```suggestion
teleservice: https://www.ghjuventu.corsica/my/dashboard
```

Tu as un espace en trop avant les `:`